_std/types.dm

```diff
@@ -160,6 +160,9 @@ var/list/list/by_cat = list()
 #define TR_CAT_JOHNBILLS "johnbills"
 #define TR_CAT_OTHERBILLS "otherbills"
 #define TR_CAT_TELEPORT_JAMMERS "teleport_jammers"
+#define TR_CAT_BURNING_MOBS "dude_on_fire"
+#define TR_CAT_HOTSPOTS "air_on_fire"
```

Comment:
```suggestion
```
delete this, and use by_type for hotspots instead of by_cat.

Reply:
Did that, works just fine too!